### DISCUSSION TITLE
After parsing, the json key has no order

### DISCUSSION THREAD

**Discussion by ybkyozy** [nlohmann/json#3003] [General]
Can the keys be ordered?

**Comment by nlohmann** [comment]
Keys are ordered alphabetically by default. See https://json.nlohmann.me/features/object_order/ for more information.

**Comment by ybkyozy** [comment]
Thank you very much.

从 Windows 版邮件发送

发件人: Niels Lohmann
发送时间: 2021年9月8日 12:33
收件人: nlohmann/json
抄送: kyozy; Author
主题: Re: [nlohmann/json] After parsing, the json key has no order (#3003)

Keys are ordered alphabetically by default. See https://json.nlohmann.me/features/object_order/ for more information.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
